Makefile

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/make -f
 
-PACKAGES_SIMTEST=$(shell go list ./... | grep '/simulation')
```

Comment:
Why was this deleted?

Reply:
It wasn't referenced anywhere in the project

`make test-sim` seems to be using `PACKAGES_SIM`:
https://github.com/osmosis-labs/osmosis/blob/aeca891286ef91e8706e1247276fabc93445ea69/Makefile#L218

Reply:
oh your right, its also unused in the SDK as well